overrides/config/ftbquests/quests/chapters/quests.snbt

```diff
@@ -97,7 +97,7 @@
 			y: 15.25d
 		}
 		{
-			click: ""
+			click: "command:/playsound minecraft:entity.creeper.primed master @p ~ ~ ~ 2 1 1"
```

Comment:
just keep in mind the commands will probably only work if the player is topped or has cheats enabled